collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -116,6 +123,18 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.main_menu);
         initToolbar();
 
+        // Enable TLS v1.2 for Android 4.2.2
+        try {
+            ProviderInstaller.installIfNeeded(getApplicationContext());
+            SSLContext sslContext;
+            sslContext = SSLContext.getInstance("TLSv1.2");
+            sslContext.init(null, null, null);
+            sslContext.createSSLEngine();
+        } catch (GooglePlayServicesRepairableException | GooglePlayServicesNotAvailableException
```

Comment:
Seems wise to separate out these exceptions into two blocks -- the GooglePlayServices errors, and then the other errors (presumably related to SSLContext).